website/docs/docs/deploy/advanced-ci.md

```diff
@@ -31,6 +31,18 @@ dbt reports the comparison differences in:
 
 <Lightbox src="/img/docs/dbt-cloud/example-ci-compare-changes-tab.png" width="85%" title="Example of the Compare tab" />
 
+### Considerations
+You can set a [event_time](/reference/resource-configs/event-time) for a model, seed, snapshot, or source to represent the actual timestamp of the event, rather than something like loading date. This configuration is important for certain features, such as [Incremental microbatch](/docs/build/incremental-microbatch) and compare changes in CI/CD workflows, where it ensures the same time-slice of data is accurately compared between your CI and production environments.
+
+When configured, `event_time` enables compare changes to:
+
+- Compare data in CI versus production for overlapping times only, reducing false discrepancies.
+- Handle scenarios where CI contains fresher data than production by using only the overlapping timeframe, which avoids incorrect row-count changes.
+- Account for subset data builds in CI without flagging filtered-out rows as "deleted" when compared with production.
```

Comment:
```suggestion
```
These first three bullets seem to be saying basically the same thing - have collapsed them down a bit.